### PR TITLE
Register member

### PR DESCRIPTION
**Pull request by DinolanNaidoo**

### CODE REVIEW

File: skietbaan_fe/src/components/ViewMembers.css [hunk 1]
```diff
@@ -7,35 +7,131 @@
...
     text-align: center;
     color: white;
   }
+
+  .username-search{
+    position: center;
+    text-align: center;
+  }
+
+  .userValue {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewMembers.css]: inconsistent indentation and please leave a space after classname and first bracket this is also inconsistent

File: skietbaan_fe/src/components/ViewMembers.js [hunk 2]
```diff
@@ -33,11 +36,28 @@ class ViewMembers extends Component {
             }).then(data => this.setState({
                 array: data
             }))
-            .catch(function() { });
+            .catch(function () { });
+    }
+
+
+    GetFilteredMembers() {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewMembers.js]: This function should look something like this 
```
fetch(BASE_URL + "/api/Features/SearchMember", {
	method: 'Get',
	headers: { 'Accept': 'application/json', 'Content-Type': 'application/json' },
}).then(function (response) {
	return response.json();
}).then(function (data) {
	return data;
}).then(data => this.setState({
	array: data
})).catch(function () { });
```

Comment (FredJEhlers) [reply skietbaan_fe/src/components/ViewMembers.js]: Also, why catch if you aren't doing anything?

File: skietbaan_fe/src/components/ViewMembers.css (hunk 1 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/ViewMembers.css]: made it look better

File: skietbaan_fe/src/components/ViewMembers.js (hunk 2 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/ViewMembers.js]: okay removed all the catches

**Pull request merged**